frontend/pages/admin/IntegrationsPage/cards/MdmSettings/MdmSettings.tsx

```diff
@@ -63,13 +63,10 @@ const MdmSettings = ({ router }: IMdmSettingsProps) => {
               turnOnMacOSMdm={navigateToMacOSMdm}
               viewDetails={navigateToMacOSMdm}
             />
-            {/* TODO: remove conditional rendering when windows MDM is released. */}
-            {config?.mdm_enabled && (
-              <WindowsMdmCard
-                turnOnWindowsMdm={navigateToWindowsMdm}
-                editWindowsMdm={navigateToWindowsMdm}
-              />
-            )}
+            <WindowsMdmCard
+              turnOnWindowsMdm={navigateToWindowsMdm}
+              editWindowsMdm={navigateToWindowsMdm}
+            />
```

Comment:
Or should I have kept the `{}` around `<WindowsMdmCard>`?

Reply:
Thanks, I'll remove them from [here](https://github.com/fleetdm/fleet/pull/15167#discussion_r1417487710) then.